spec/models/openingtime_spec.rb

```diff
@@ -1,8 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe Openingtime, type: :model do
+
+  before(:each) do
```

Comment:
```suggestion
  before :each do
```

Reply:
I'm not sure, whether that's the linter's desire, but in my opinion the linter should be pleased before merging